docs/abi.md

```diff
@@ -112,6 +112,12 @@ imports](https://webassembly.github.io/spec/core/syntax/modules.html#imports)
     *   arg 1: Source buffer size in bytes
     *   return 0: Channel handle, or zero if not found.
 
+-   `random_get: (i32, i32) -> i64`: Fill a buffer with random bytes.
```

Comment:
Do you have some use case already in mind for this? I was thinking that we would not need to expose randomness to wasm modules, ideally all the crypto operations should be performed by the trusted runtime anyways.

Reply:
No specific use case yet, but it just seems inevitable that we're going to want randomness inside Node code eventually (e.g. what if we want to generate a Swiss number to use as a remote capability?).